templates/views/views-view-unformatted--search.html.twig

```diff
@@ -0,0 +1,14 @@
+{#
+ # Remove wrapping div from all containers by default.
+ # For the default template, see `web/core/themes/stabel9/templates/views/views-view-unformatted.html.twig`.
+ #}
+
+{% if title %}
+  <h3>{{ title }}</h3>
+{% endif %}
+
+{{ drupal_block('ys_search_form_block') }}
+
+{% for row in rows %}
+  {{- row.content -}}
+{% endfor %}
```

Comment:
Again, this should be a (probably super basic) Organism in the component library. You're using an `<h3>` here, when it should be an `<h2>` ... actually... looking at the page, I don't think the view title is ever showing... so we probably don't even need that markup... and if that's the case this may not need a component.

The only reason you might want one, is if you use one of the "list" organisms - like the card list organism to get the page gutters, and the divider lines between items... Okay, I'm thinking out loud here 😂  There definitely needs to be a component wrapping the rows because we don't want the content touching the edge of the browser.

And it looks like the pagetitle block needs to reference something so that it doesn't touch the edge as well. I think that block is only visible on the search page right now, so this is the first time we've seen it actually used.

Reply:
Same for this one - different ticket (YALB-96) will take care of this one.

Reply:
Was deleted in [PR-68](https://github.com/yalesites-org/atomic/pull/68)